libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -505,7 +505,7 @@ void NavEKF2_core::readGpsData()
     // do not accept data at a faster rate than 14Hz to avoid overflowing the FIFO buffer
     const AP_GPS &gps = AP::gps();
     if (gps.last_message_time_ms() - lastTimeGpsReceived_ms > 70) {
-        if (gps.status() >= AP_GPS::GPS_OK_FIX_3D) {
+        if (gps.status() >= gps.status_arm_min()) {
```

Comment:
We should accept GPS data below the arm min

Reply:
Done